drag_and_drop_v2/public/js/drag_and_drop_edit.js

```diff
@@ -119,11 +118,18 @@ function DragAndDropEditBlock(runtime, element) {
                             _fn.build.form.zone.add();
                         })
                         .on('click', '.remove-zone', _fn.build.form.zone.remove)
+                        .on('input', '.zone-row input', _fn.build.form.zone.changedInputHandler)
                         .on('click', '.target-image-form button', function(e) {
                             e.preventDefault();
 
-                            _fn.data.targetImg = $('.target-image-form input', element).val();
-                            _fn.$target.css('background', 'url(' + _fn.data.targetImg + ') no-repeat');
+                            var new_img_url = $('.target-image-form input', element).val();
+                            _fn.data.targetImg = new_img_url;
+                            // We may need to 'expand' the URL before it will be valid.
+                            // e.g. '/static/blah.png' becomes '/asset-v1:course+id/blah.png'
+                            var handlerUrl = runtime.handlerUrl(element, 'expand_static_url');
+                            $.post(handlerUrl, JSON.stringify(new_img_url), function(result) {
+                                _fn.build.$el.targetImage.attr('src', result.url);
+                            }).error(function(a, b, c) { console.log(a, b, c); });
```

Comment:
@bradenmacdonald This a leftover from manual testing? :) If not, it would be good to use more explicit variable names for `a`, `b`, `c` here.